components/brave_wallet_ui/common/trezor/trezor_bridge_keyring.ts

```diff
@@ -97,6 +121,34 @@ export default class TrezorBridgeKeyring extends EventEmitter {
     return ''
   }
 
+  private normalize (buf: any) {
+    return bufferToHex(buf).toString()
+  }
+
+  private prepareTransactionPayload = (path: string, txInfo: TransactionInfo, chainId: string): SignTransactionCommandPayload => {
+    const txParams = {
+      nonce: txInfo.txData.baseData.nonce,
+      gasPrice: txInfo.txData.baseData.gasPrice,
+      gasLimit: txInfo.txData.baseData.gasLimit,
+      to: txInfo.txData.baseData.to,
+      value: txInfo.txData.baseData.value,
+      data: Buffer.from(txInfo.txData.baseData.data)
+    }
+    const tx = new Transaction(txParams)
+    return {
+      path: path,
+      transaction: {
+        to: this.normalize(tx.to),
+        value: this.normalize(tx.value),
+        data: this.normalize(tx.data).replace('0x', ''),
```

Comment:
it's current API doesn't need to strip 0x

Reply:
done